ci/docker/linux-apt-docs.dockerfile

```diff
@@ -55,7 +57,10 @@ RUN apt-get update -y && \
         r-base=${r}* \
         rsync \
         ruby-dev \
-        wget && \
+        wget \
+        libasound2-dev && \
```

Comment:
Could you keep this list in alphabetical order?

Reply:
yup! sorry. fixed!